torch/quantization/quantize.py

```diff
@@ -333,3 +333,26 @@ def swap_module(mod, mapping):
         if type(mod) in mapping:
             new_mod = mapping[type(mod)].from_float(mod)
     return new_mod
+
+def dump_tensor(mod, target_dict, prefix=""):
```

Comment:
I'm not sure this function is helpful beyond state_dict and manual manipulation.

Also it's name is very non-descriptive - what does 'dump_tensor' mean if I just see it?

Here's a better idea - we can implement a function `all_observers_dict(module)` that acts like `module.state_dict()` but returns you the named dictionary of all observers. Then you can just call that function and write a one-liner that iterates over observers and pulls out tensor values